### ISSUE TITLE
Gnome Terminal?

### ISSUE THREAD

**Issue by KingYes** [hukl/Smyck-Color-Scheme#16]
Hey..
Can you create scheme for Gnome GTK3 terminal?

BTW, which font you use?

Thanks :]

**Comment by bclouser** [comment]
Yeah! I love this scheme on my mac, it would be ideal to use on linux as well. Or if i knew how to port it...

**Comment by vcavallo** [comment]
I've created [this gnome-smyck installation repo](https://github.com/vcavallo/gnome-terminal-colors-smyck), based off of [this work](https://github.com/Anthony25/gnome-terminal-colors-solarized). Read the readme and run the installer. It's working for me, and i'm very happy about it :)

![image](https://cloud.githubusercontent.com/assets/3201135/12532907/6b727818-c1ed-11e5-881e-d2ef028282e8.png)

Let me know if and how you'd like this incorporated into your main repo and I'll submit a pull request. Enjoy!

**Comment by bclouser** [comment]
Oh man, that's cool! Well done!

Sadly, since commenting on this i have become a "terminator" user so I don't use gnome terminal much anymore. I found a smyck terminator config file somewhere and have been using it ever since.
